### ISSUE TITLE
Launcher Stuck on Loading Screen (Error while obtaining machine id)

### ISSUE THREAD

**Issue by InsertColon** [killpowa/Twickt-Launcher#1428]
**Describe the bug**
The **GDLauncher** app is **stuck** on the first loading screen after the first installation

**To Reproduce**
1. Go to 'https://gdevs.io/' or 'https://github.com/gorilla-devs/GDLauncher'
2. Scroll down to 'Download'
3. Click on 'Download for Windows' or 'here'
4. Open the GDLauncher-win-setup.exe
5. Install GDLauncher
6. Open GDLauncher
7. See error in the console with an infinite loading screen

**Expected behavior**
The Launcher should have loaded and should be working

**Possible solutions**
None found

**Screenshots**
![image](https://user-images.githubusercontent.com/110725584/183248514-4d7a1223-5396-41e9-8556-eda6706e3f37.png)
![image](https://user-images.githubusercontent.com/110725584/183248523-4f8cc9a2-4324-417f-b5fe-070a56ce25af.png)

**Operating System:**
- OS: Windows
- Version: 10
- Java Version: 8
- Minecraft Version: Not important
- Mod Loader: Not important

**Additional context**
So, basically I went to the GDLauncher website, I downloaded it, I installed it but when I opened it, It was stuck on the loading screen even after waiting 24 hours loading. Then I tried reinstalling it more than 8 times from github and gddevs.io even the legacy version, I tried the Java Development Kit 8 hotspot and Java from Java.com even let the GDLauncher Install Java itself but nothing worked even after deleting nearly if not every Java path in my computer.

**Comment by Eskaan** [comment]
The specific error above is caused by the specified Java executable not being found.

It could help deleting the %AppData%\gdlauncher_next folder. Be careful, your instances are usually kept in this folder.

**Comment by InsertColon** [comment]
ok thanks I'll try it but this may not work

**Comment by InsertColon** [comment]
@Eskaan It didn't work I also tried deleting the cache but it didn't work aswell

**Comment by InsertColon** [comment]
do you have any other advice as how i can fix this error ?

**Comment by Eskaan** [comment]
There should be a different error message now.

**Comment by InsertColon** [comment]
yeah it's a bit longer but the same error tho

**Comment by InsertColon** [comment]
![image](https://user-images.githubusercontent.com/110725584/183294224-95af3b0c-a2b6-4c6e-b30f-946e7d51a628.png)
here is the logs

**Comment by InsertColon** [comment]
@Eskaan this is what i got

**Comment by Eskaan** [comment]
Okay, it's the same. Bug-Fatal then. I never saw this before and can't reproduce it.

**Comment by InsertColon** [comment]
@Eskaan do you know someone that can try to help me ?

**Comment by Eskaan** [comment]
@Ladvace may be able to fix the issue.

**Comment by Ladvace** [comment]
could you try running gdl as admin and check if you still get the error?

**Comment by InsertColon** [comment]
yeah i tried @Ladvace but i'm gonna try it gonna wait (sorry for being late)

**Comment by InsertColon** [comment]
It's the same error 
![image](https://user-images.githubusercontent.com/110725584/183304639-22cea812-462f-4c4e-913a-1daa38e6187e.png)

**Comment by InsertColon** [comment]
Oh yeah my Jdk path is C:\Program Files\Eclipse Adoptium and not C:\Program Files\Eclipse Foundation is this a problem ? and if yes do I need to rename the file or something

**Comment by Ladvace** [comment]
I think it's a bug with a library we use to get the machine id, not sure why you get the error tho, if you download gd and let it auto install java do you still get the error?

**Comment by InsertColon** [comment]
yeah i still get it but i'll try it again and tell you if it worked

**Comment by InsertColon** [comment]
same error @Ladvace I am wondering if i'm uninstalling java correctly (I uninstall it from the control panel)

**Comment by InsertColon** [comment]
![image](https://user-images.githubusercontent.com/110725584/183397964-480e3388-25af-4eb8-88aa-0448fe1cfcc4.png)
I found the main logs in C:\Users\User\AppData\Roaming\gdlauncher\logs and it's just looping this
